src/backend/plugins/pacrunner-duktape/meson.build

```diff
@@ -5,13 +5,21 @@ if get_option(plugin_name)
 duktape_dep = dependency('duktape')
 m_dep = cc.find_library('m', required : false)
 
+# Required for getnameinfo() and others.
+if host_machine.system() == 'sunos'
+  network_dep = [ cc.find_library('socket', required: false) ]
```

Comment:
It should be `socket_dep` and does not need an if protection i think

Reply:
Actually you can just add it to `px_backend_deps` directly. Should be a one liner